File: Lib/FPVR/Interfaces/ISpeedInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace FPVR
{
    /// <summary>
    /// The interface for a class that provides speed information.
    /// </summary>
    public interface ISpeedInfo
    {
        /// <summary>
        /// Gets the forward speed in meters per second.
        /// </summary>
        double ForwardSpeed { get; }
    }
}

File: Android/FPVR/app/src/main/java/com/solersoft/fpvr/fpvrlib/ISpeedInfo.java
package com.solersoft.fpvr.fpvrlib;

/**
 * The interface for a class that provides speed information.
 */
public interface ISpeedInfo extends IVehicleService
{
    //region Public Properties
    /**
     * Gets the forward speed in meters per second.
     */
    public double getForwardSpeed();
    //endregion
}
